netlify.toml

```diff
@@ -34,8 +34,6 @@ package = "netlify-plugin-checklinks"
     "public/docs/components/pipelines/fonts/",
     "public/docs/components/pipelines/legacy-v1/fonts/",
 
-    "public/docs/components/katib/",
-    "public/docs/components/kfserving/",
```

Comment:
Did we update the KFServing docs ?

Reply:
No. It seems like the broken link was fixed in another PR, but KFServing was not removed from the white list.
Since there are no more broken links (we can confirm it in the CI of this PR), it should be removed.